qiita_pet/templates/analysis_results.html

```diff
@@ -12,6 +12,19 @@
 
 <div class="row">
   <h1>&nbsp;Analysis: <u>{{aname}}</u></h1>
+  {% if status == "altered_data" %}
+    <h2 style="color:red">UNDERLYING DATA FOR THIS ANALYSIS HAS CHANGED. CONSIDER RE-RUNNING THIS ANALYSIS.</h2>
+    {% for study, change in changes %}
+    <p><b>study</b><br/>
+      {% for c in change %}
+        {{c}}<br/>
+      {% end %}
+      </p>
+    {% end %}
+    {% if changes == {} %}
```

Comment:
it did for delete updates, but since that's not a thing any more this check is no longer needed.